### ISSUE TITLE
[Enhancement] Unsuspend adjacent tabs.

### ISSUE THREAD

**Issue by awiebe** [deanoemcke/thegreatsuspender#639]
It would be nice if tabs adjacent to a current tab could be unsuspended, and marked as not suspendable, so instead of reloading tabs when closing many tabs in a window sequentially, the tabs have already been loaded in.

**Comment by deanoemcke** [comment]
I'm not 100% sure what your work flow is with this.
Firstly, this is assuming that a person works with tabs in a sequential order? So that most likely, after finishing with a tab, they will then proceed to the next tab (to the right?).

So the use case is, you have opening a lot of tabs a long time ago and they have all suspended. Now you go to one and unsuspend it, and you would like the one 'after' that to also unsuspend so that it is already loaded when you want to look at it?

Just trying to clarify the feature request. Thanks.

**Comment by awiebe** [comment]
@deanoemcke
Yes that would be the workflow, though just un general when you close a chrome tab the browser selects the adjacent one next, so basically it's always a good idea to keep the adjacent tab loaded, that way there is no downtime unless you close a lot of tabs really quickly.

**Comment by johnyoonh** [comment]
Yes this feature would be very handy, as the most user's workflow is closing a tab and the browser switches over to the previous tab. Having this feature and having it warmed up would save a lot of time as some may never see unsuspend delay.
